addon/data/shoot-panel.js

```diff
@@ -173,7 +173,7 @@ class ShootPanel extends React.Component {
   }
 
   onLinkClick(e) {
-    self.port.emit("openLink", this.props.shot.viewUrl, "normal");
+    self.port.emit("openLink", this.props.shot.viewUrl, this.props.loadReason, "normal");
```

Comment:
Actually I'm confused about the parameters to openLink, and the code doesn't make it clear to me either...